### DISCUSSION TITLE
真就最后的疯狂了么？从中午的崩三到晚上的崩铁少说也拉黑了上百个号，啊b还在给我推节奏视频

### DISCUSSION THREAD

**Discussion by cyq20091029** [share121/inter-knot#2482] [其他]
<img width="345" alt="截屏2024-08-17 04 31 01" src="https://github.com/user-attachments/assets/9a53245f-c0d7-4e95-bedd-1edab5442fd9">
希望9月新规能净化一下环境

**Comment by nostalgiatan** [comment]
希望吧，u1s1在视频平台里b站还是比音符慢脚好多了，绳网的推广应该只有b站，从某种意义上也可以算B站帮我们过滤掉了一部分人，绳网登录的难度也刷掉了一部分人，目前绳网的环境也算可以。

**Comment by whoisfucker** [comment]
用这个油猴脚本来屏蔽B站标签和点赞比例过滤视频，我屏蔽后，主页的视频就屏蔽了一半：
https://www.bilibili.com/video/BV12i4y1e73B/?spm_id_from=333.880.my_history.page.click&vd_source=c70ec12f39016f2c5b0dd55e4597ca1d

**Reply by cyq20091029** [reply]
感谢大佬

**Comment by Dave-12138** [comment]
我坚持答题当硬核会员不是因为我喜欢闪电标
而是普通会员黑名单不够用<img src="https://bbs-static.miyoushe.com/static/2023/12/22/4da6b5aa297be428e0c4c478c33d11db_1621822785592350465.png" alt="月下-威胁" width="75" >

**Comment by 00zzxx** [comment]
不过九月那个能不能有用难说吧，只能说不抱太大期待了<img src="https://bbs-static.miyoushe.com/static/2024/01/31/5cc3b07992e564dc8389bb6403dfc115_1913269177686026716.png" alt="米沙 自闭" width="75" >
